x-pack/metricbeat/module/istio/pilot/_meta/fields.yml

```diff
@@ -0,0 +1,84 @@
+- name: pilot
+  type: group
+  description: >
+    Contains statistics related to the Istio pilot service
+  release: beta
+  fields:
+    - name: xds.count
+      type: integer
+      description: >
+        Count of concurrent xDS client connections for Pilot.
+    - name: xds.pushes
+      type: integer
+      description: >
+        Count of xDS messages sent, as well as errors building or sending xDS messages
+        for lds, rds, cds and eds.
+    - name: xds.push.time.sec
+      type: long
+      description: >
+        Total time in seconds Pilot takes to push lds, rds, cds and eds.
+    - name: xds.eds.instances
+      type: object
+      description: >
+        Instances for each cluster, as of last push. Zero instances is an error.
+    - name: xds.push.context.errors
+      type: integer
+      description: >
+        Number of errors (timeouts) initiating push context.
+    - name: xds.internal.errors
+      type: integer
+      description: >
+        Total number of internal XDS errors in pilot.
+    - name: conflict.inbound.listener
+      type: integer
+      description: >
+        Number of conflicting inbound listeners.
+    - name: conflict.outbound.listener.http.over.current.tcp
```

Comment:
Same as above, I would suggest grouping this like `conflict.listener.outbound....`

Same for the rest of the `listener` related events.